src/Microsoft.Health.Fhir.SqlServer/Features/Search/SearchOptionsExtensions.cs

```diff
@@ -37,5 +37,18 @@ public static (Core.Models.SearchParameterInfo, SortOrder) GetFirstSupportedSort
 
             return (null, SortOrder.Ascending);
         }
+
+        /// <summary>
+        /// A helper method to determain if a supported sort param exists.
+        /// </summary>
+        /// <param name="searchOptions">The SearchOptions context</param>
+        /// <returns>True if exists, false otherwise.</returns>
+        public static bool SupportedSortParamExists(this SearchOptions searchOptions)
+        {
+            Core.Models.SearchParameterInfo searchParamInfo = GetFirstSupportedSortParam(searchOptions).Item1;
```

Comment:
> Core.Models.SearchParameterInfo searchParamInfo = GetFirstSupportedSortParam(searchOptions).Item1 [](start = 11, length = 98)

super nit:
There is `_` discard variable in C# 7.0 which allow you to write code like this:
`var (searchParamInfo,_) =  GetFirstSupportedSortParam(searchOptions);`
which I find a bit clear.